pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -225,7 +221,8 @@ in rec {
 
   bareFrameworks = stdenv.lib.mapAttrs framework (import ./frameworks.nix {
     inherit frameworks libs;
-    inherit (pkgs.darwin) CF cf-private libobjc;
+    inherit (pkgs.darwin) cf-private libobjc;
+    CF = CoreFoundation;
```

Comment:
@copumpkin @LnL7 
How do you all feel about this? It gives an "impure" CF to already impure frameworks. This should improve our support for these frameworks, at the expense of less purity in our CF.

I'll probably just leave it here for now with the hope being that Apple updates CF in the next year or so.